source/index.html.md

```diff
@@ -43,7 +43,7 @@ The valid JWT token must be supplied as part of the header to access API's all o
 
 <pre>curl --request GET \
   --url https://api-stg3.snapchance.no/profile \
-  --header 'authorization: Bearer $SNAPCHANCE_API_TOKEN`'</pre>
+  --header 'authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0eXBlIjoic2Vzc2lvbiIsImV4cCI6MTYxNzM0MzI2MywicGxheWVyX2lkIjoyLCJlbWFpbCI6IjU5Y2MifQ.fUyn-pli3fe6knErVCUj6WQ_5NbRR6_RMqtaB7Q21t4'</pre>
```

Comment:
Was this token meant to be updated to the hardcoded value?

Reply:
Good catch.

The original idea was to have all example requests showing `$SNAPCHANCE_API_TOKEN`. 

Unfortunately it doesn't work. So this is how it's being generated from the current swagger file. So I think this change makes sense ATM.

Reply:
Ok thanks!